Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -261,13 +280,17 @@ class BoolSetting: Setting, FeatureFlaggable {
         return statusText
     }
 
+    public lazy var control: UISwitch = {
+        let control = UISwitch()
+        control.accessibilityIdentifier = prefKey
+        control.addTarget(self, action: #selector(switchValueChanged), for: .valueChanged)
+        return control
+    }()
```

Comment:
we sholud be using our `.build` pattern here. see `public static func build<T: UIView>(_ builder: ((T) -> Void)? = nil) -> T {` for more info. :)

Reply:
Weirdly enough, if I add `translatesAutoresizingMaskIntoConstraints = false` either on the spot or with `.build` the UI breaks. Is it maybe because the switch is an accessory view to the cell and it needs to be `true`?

![Screenshot 2023-03-10 at 2 02 03 PM](https://user-images.githubusercontent.com/26011662/224311119-d587fddd-5dc3-4201-810a-67e52fc8eaca.jpeg)

Reply:
Weird! Ok then, let's leave it as is, and we can investigate further when there's time.